test.sh

```diff
@@ -3,17 +3,16 @@
 PWD=`pwd`
 
 cd test_project
-python manage.py test
+poetry run python manage.py test
 EXIT1=$?
 
 EXIT2=0
 while getopts ":p" opt; do
     case $opt in
         p)
-            pip install psycopg2-binary
             cd ../test_project_postgres
-            python manage.py migrate
-            python manage.py test --no-input
+            poetry run python manage.py migrate
+            poetry run python manage.py test --no-input
```

Comment:
If you run this script with `poetry run  test.sh`, you won't need to change this script to know about `poetry`.